### PR TITLE
Tab Code block

### PR DESCRIPTION
**Pull request by BaskarMitrah**
## Description

Feat: Added code block inside the tab block

## How Has This Been Tested?

https://code-tab-block-spectrum-less--adobe-io-website--adobe.hlx.page/test/code-tab-block

## Screenshots (if appropriate):

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by melissag-ensemble — COMMENTED** [review]
This block seems very similar to one that already exists: https://github.com/adobe/adobe-io-website/pull/206/files. Should we combine it somehow? What ticket is this for - perhaps that'll give us more context.

Or maybe this is ok... @timkim what do you think?

**Comment by BaskarMitrah** [comment]
> This block seems very similar to one that already exists: https://github.com/adobe/adobe-io-website/pull/206/files. Should we combine it somehow? What ticket is this for - perhaps that'll give us more context.
> 
> @timkim what do you think?

**Comment by BaskarMitrah** [comment]
@melissag-ensemble, Here, I mentioned the ticket [https://jira.corp.adobe.com/browse/DEVSITE-1081]

**Comment by melissag-ensemble** [comment]
I'm leaning towards keeping this (and throwing away https://github.com/adobe/adobe-io-website/pull/206/files) because they're very similar but this handles both tab orientations

**Comment by timkim** [comment]
I think we keep both. They are technically different blocks on our site. The default code block is more for documentation purposes and this tab code block is more used on the marketing pages. It might be worthwhile to import in the code block into here then? I'm not sure about the implementation of the code block and how easily we can import the functions into this one. What do you think @melissag-ensemble ?

**Comment by melissag-ensemble** [comment]
Sounds good, we can keep both. In terms of code reuse,

We can probably extract these lines: https://github.com/adobe/adobe-io-website/blob/fresh-spectrum-less/hlx_statics/blocks/codeblock/codeblock.js#L54-L60

So it can be reused in: https://github.com/adobe/adobe-io-website/blob/fa76150db9db9b0659d7d85f7768643036cf1371/hlx_statics/blocks/tab/tab.js#L74-L86

**Comment by BaskarMitrah** [comment]
Okay @melissag-ensemble, We will reuse the code block. I'll change it and let you know. Thanks!!

File: hlx_statics/styles/styles.css [hunk 1]
```diff
@@ -11,6 +11,7 @@
  */
 
  @import url('./spectrum/spectrum.min.css');
+ @import url('./prism.css');
```
Comment (melissag-ensemble) [inline hlx_statics/styles/styles.css:14]: This loads prism css all the time, I wonder if we should only import this within code-related blocks (i.e. codeblock, code, tab) for optimization @timkim @BaskarMitrah what do you think?

Comment (BaskarMitrah) [reply hlx_statics/styles/styles.css:14]: Yes @melissag-ensemble, It would be nice!!!. Shall I import the link into these two components only? @timkim

Comment (timkim) [reply hlx_statics/styles/styles.css:14]: Yes - we should only load it when we need it.

**Review by melissag-ensemble — APPROVED** [review]

**Comment by melissag-ensemble** [comment]
@BaskarMitrah please resolve conflict

**Pull request merged**